packages/scene-composer/src/components/panels/SceneRulesPanel.tsx

```diff
@@ -82,7 +82,7 @@ const SceneRuleMapExpandableInfoSection: React.FC<React.PropsWithChildren<IScene
           {
             label: intl.formatMessage({ defaultMessage: 'Expression', description: 'Input field label' }),
             control: (item, itemIndex) => (
-              <Input
+              <Textarea
```

Comment:
What's the reason to moving from Input to TextArea?

Reply:
Customers would have to paste in a very long propertyName (e.g. 'Property1111-2222-3333-4444-5555') into the rule expression and it would take up all of the text input field, preventing them from seeing the rest of their rule expression (e.g. "== 50").

Reply:
Is there a limit on the length of the property name?

Reply:
And also you should write unit test case to test your feature with different values.

Reply:
This needs to addressed later :)